pkg/controllers/provisioningv2/rke2/rkebootstrap/controller.go

```diff
@@ -1,10 +1,10 @@
-package bootstrap
```

Comment:
I would prefer we do not rename packages at this time. It makes maintenance very difficult. We'll be much better off doing this once we've stabilized our code base.